presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -2369,6 +2372,11 @@ public void testGroupedExecution()
                             "WITH (bucket_count = 13, bucketed_by = ARRAY['keyD']) AS\n" +
                             "SELECT orderkey keyD, comment valueD FROM orders CROSS JOIN UNNEST(repeat(NULL, 2))",
                     30000);
+            assertUpdate(
+                    "CREATE TABLE test_window\n" +
```

Comment:
nit: Maybe call it `test_grouped_window`? 

Also remember to use 13 buckets. Since 13 is co-prime with 4 (number of tasks), thus it tests unbalanced assignment ;)